openfl-workspace/torch_cnn_histology_gramine_ready/src/pthistology_inmemory.py

```diff
@@ -21,8 +21,14 @@ def __init__(self, data_path, batch_size, **kwargs):
         """
         super().__init__(batch_size, random_seed=0, **kwargs)
 
+        try:
+            int(data_path)
+        except:
+            raise ValueError("Expected `%s` to be representable as `int`.", data_path)
```

Comment:
Same comment here, and for the other `ValueError` instances throughout the PR

Reply:
Addressed in ca90679af5ca4788e1bc8886dda550dd34f57a0d.